cognite/powerops/resync/core/main.py

```diff
@@ -31,6 +31,7 @@
 from cognite.powerops.resync.diff import FieldDifference, ModelDifference, ModelDifferences
 from cognite.powerops.resync.models.base import AssetModel, CDFFile, CDFSequence, DataModel, Model, SpaceId
 from cognite.powerops.resync.models.v2.powerops_models import DataModelLoader
+from cognite.powerops.resync.v2.shop_to_assets import PowerAssetImporter  # type: ignore[attr-defined]
```

Comment:
**question:** why are we ignoring this mypy check?

Reply:
No longer necessary after your suggestion on mypy ignore.